changelog.txt

```diff
@@ -3,6 +3,7 @@
 = 2.4.0 - 2021-xx-xx =
 * Update - Improve the Connect Account page.
 * Update - Base UI components and their styling.
+* Fix - Error when purchasing free trial subscriptions.
```

Comment:
Since we're going to do a release for this fix, could you create a 2.3.2 header here and in the readme.txt?

The date can still be `2021-xx-xx`, the release tool will add that in for us.

Reply:
Done. I put it below 2.4.0. Not sure if that'd work.